src/fleet/azext_fleet/_help.py

```diff
@@ -23,6 +23,30 @@
           type: string
           short-summary: Prefix for hostnames that are created. If not specified, generate a hostname using the
                          managed cluster and resource group names.
+        - name: --enable-private-cluster
+          type: bool
+          short-summary: Prevents the Fleet's API server from be publicly accessed.
+        - name: --enable-vnet-integration
+          type: bool
+          short-summary: Enables the Fleet to communicate on a virtual network.
+        - name: --apiserver-subnet-id
+          type: string
+          short-summary: The sub network that the Fleet's API server will use.
+        - name: --agent-subnet-id
+          type: string
+          short-summary: The sub network that the Fleet's agents will use.
```

Comment:
verbatim from swagger spec: "The ID of the subnet which the Fleet hub node will join on startup. If this is not specified, a vnet and subnet will be generated and used."